docs/docs/monitoring/load-testing-guidelines.mdx

```diff
@@ -17,6 +17,29 @@ In our tests we used the Rasa [HTTP-API](https://rasa.com/docs/rasa/pages/http-a
 | Up to 50,000             |         6vCPU                                |      16 GB    |
 | Up to 80,000             |         6vCPU, with almost 90% CPU usage     |      16 GB    |
 
+:::info This is the most optimal AWS setup tested on EKS with
```

Comment:
Hi @sanchariGr ,
These recommendations are too AWS specific, I think we should rewrite them in a more general way.